packages/data/src/world/mq/bottom_of_the_well_mq.yml

```diff
@@ -18,39 +18,21 @@
 "Bottom of the Well Main":
   dungeon: BotW
   exits:
-    "Bottom of the Well Wallmaster Main": "soul_wallmaster && (can_play_zelda || (has_explosives_or_hammer && has_hover_boots))"
-    "Bottom of the Well Wallmaster Basement": "soul_wallmaster"
-    "Bottom of the Well Wallmaster Pit": "soul_wallmaster && (can_hit_triggers_distance || can_boomerang || has_explosives || can_hookshot)"
-    "Bottom of the Well": "true"
+    "Bottom of the Well": "is_child"
+    "Bottom of the Well Main Room Center": "event(BOTW_MQ_CENTER_GATES_LOWERED)"
+    "Bottom of the Well Main Room Center Switch Corner": "has_explosives_or_hammer"
+    "Bottom of the Well Basement": "true"
+    "Bottom of the Well Coffin Room": "(event(BOTW_MQ_DRAIN_WATER) || can_dive_small) && small_keys_botw(2)"
```

Comment:
Surely hook anywhere can do this too, possibly not even anywhere and just normal hook.

Reply:
added hook anywhere.

Reply:
Added.